webapp/src/helpers/__tests__/meetingsHelper.ts

```diff
@@ -0,0 +1,87 @@
+import { compareMeetingNotes } from '../meetingsHelper';
+import { MeetingNote } from '../../types/api';
+
+describe('meetingsHelper', () => {
+  describe('compareMeetingNotes', () => {
+    it('should sort non-null participants first', () => {
+      expect(
+        compareMeetingNotes(
+          { agenda_owning_participant_id: 1 } as MeetingNote,
```

Comment:
Should the arguments passed into the `compareMeetingNotes` have the same structure/properties as the MeetingNote object type?

Reply:
I can think of more cons for filling out the objects than pros. For example, only including properties that matter for the specific example makes it clearer how the example works. It also causes the test to fail in the event that the implementation changes to consider those other properties before it takes the specified ones into account. I'd rather have false negatives than false positives, and I prefer less code over more code.